src/App.css

```diff
@@ -1,32 +1,7 @@
-.App {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  margin: 0 auto;
-  text-align: center;
-}
-
 .FAQ {
   background-color: aliceblue;
 }
 
-h1 {
-  margin: 40px auto;
-  text-align: center;
-}
-
-ul {
-  margin-top: 40px;
-}
-
-li {
-  color: orange;
-  display: inline-flex;
-  list-style: none;
-  padding: 0 30px;
-  width: 120px;
-}
-
 a:link {
   text-decoration: none;
 }
```

Comment:
❓ Are we still using this file for anything? If not, can you remove it?

Reply:
I removed the file but haven't found a way to style this with native material ui props yet.  Creating an issue for this to come back to.